docs/src/How_to_implement_a_new_algorithm.md

```diff
@@ -10,43 +10,49 @@ function _run(policy::AbstractPolicy,
         stop_condition::AbstractStopCondition,
         hook::AbstractHook,
         reset_condition::AbstractResetCondition)
-
+    push!(hook, PreExperimentStage(), policy, env)
```

Comment:
If you add the hooks in this loop, then you should remove the "hooks were removed ..." above.